src/chess/net/net_trivial.rs

```diff
@@ -0,0 +1,440 @@
+use crate::chess::chess_game::{ChessGame, ChessPosition, BOARD_SIZE};
+use crate::game::common::{GamePosition, IGame};
+use crate::game::mcts::ValueFunction;
+use crate::game::net::{self, TwoHeadedNetBase};
+use once_cell::sync::Lazy;
+
+pub struct TrivialNet {}
+impl TrivialNet {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl ValueFunction<ChessGame> for TrivialNet {

```

Comment:
can you add a link to Leela's implementation?
btw i wonder what leela use this heuristic for?

Reply:
I think they only used it to test the MCTS implementation, separating it from the network, and I don't think it's used for the main engine